crates/goose-server/src/routes/configs.rs

```diff
@@ -154,15 +155,23 @@ pub async fn get_config(
 ) -> Result<Json<GetConfigResponse>, StatusCode> {
     verify_secret_key(&headers, &state)?;
 
-    // Fetch the configuration value. Right now we don't allow get a secret.
-    let config = Config::global();
-    let value = if let Ok(config_value) = config.get_param::<String>(&query.key) {
-        Some(config_value)
-    } else {
-        std::env::var(&query.key).ok()
+    let value = match query.key.as_str() {
+        // Special handling for model-limits
+        "model-limits" => {
+            let limits = ModelConfig::get_all_model_limits();
+            Some(serde_json::to_string(&limits).unwrap_or_default())
```

Comment:
Above we throw an 500 if we can't get valid json, but here we return an empty object `{}`, think it should be consistent either way?